### PR TITLE
Create new menu with extended options

### PR DESCRIPTION
**Pull request by Tamirsh507**
Added the option to create the rsa keys and verify the public key

### CODE REVIEW

File: fb_recovery_tool.py
Comment (amirlevygit) [inline fb_recovery_tool.py:1]: rename to fireblocks_key_backup_and_recovery.py

File: utils/public_key_verification.py
Comment (amirlevygit) [inline utils/public_key_verification.py:1]: Update readme with the new file usage and require python 3.9 (and above?)

File: fb_recovery_tool.py
Comment (Tamirsh507) [reply fb_recovery_tool.py:1]: Renamed to what Tomer requested

File: utils/public_key_verification.py
Comment (Tamirsh507) [reply utils/public_key_verification.py:1]: Updated

**Review by cblokh — APPROVED** [review]

**Pull request merged**